ts/features/messages/store/reducers/detailsById.ts

```diff
@@ -50,6 +56,29 @@ export const detailsByIdReducer = (
         [action.payload.id]: pot.noneError(error.message || "UNKNOWN")
       };
     }
+    case getType(populateStoresWithEphemeralAarMessageData): {
+      const { iun, markDown, pnServiceID, subject } = action.payload;
+      const messageData: UIMessageDetails = {
+        hasRemoteContent: true,
+        hasThirdPartyData: true,
+        id: iun,
+        markdown: markDown,
+        serviceId: pnServiceID,
+        subject
+      };
+      return {
+        ...state,
+        [iun]: pot.some(messageData)
+      };
+    }
+    case getType(terminateAarFlow):
+      const newState = _.pickBy(state, (value, _key) =>
```

Comment:
I think that using a placeholder in the markdown to identify the fake message to remove is not a good idea. We should remove the message that has been created using it's id. What about adding an optional id to the `terminateAarFlow`-action's payload and using it to identify the appropriate message?